packages/commerce-server/package.json

```diff
@@ -15,8 +15,7 @@
     "test": "jest"
   },
   "jest": {
-    "preset": "@skillrecordings/scripts/jest/node",
-    "testEnvironment": "node"
+    "preset": "@skillrecordings/scripts/jest/node"
   },
```

Comment:
Getting all of the uuid dependencies on v9 and then allowing commerce-server jest to use the default `testEnvironment` fixes the ESM/Jest/UUID issue.